Lib/pprint.py

```diff
@@ -342,6 +342,32 @@ def _pprint_mappingproxy(self, object, stream, indent, allowance, context, level
 
     _dispatch[_types.MappingProxyType.__repr__] = _pprint_mappingproxy
 
+    def _pprint_simplenamespace(self, object, stream, indent, allowance, context, level):
+        write = stream.write
+        cls_name = object.__class__.__name__
+        if cls_name == "SimpleNamespace":
```

Comment:
Why is this case necessary?  Isn't it always going to be 'namespace' (never 'SimpleNamespace')?

Reply:
Because this is how `SimpleNamespace.__repr__` works.

Reply:
I _think_ we can actually leave out the condition and just always use `cls_name = 'namespace'`.